### DISCUSSION TITLE
how to get the coding script of my ASReview project

### DISCUSSION THREAD

**Discussion by nadgheni** [asreview/asreview#1340] [Q&A]
hi!
I`m medical doctor and I`m new in the coding world!
I`ve used ASReview for 2 projects but now I want to get the coding script of this  2 projects (on Python)
Is it possible ? 
how to do it?
Thanks for your help

**Comment by Rensvandeschoot** [comment]
You can [export ](https://asreview.readthedocs.io/en/latest/manage.html#export-project)an `.asreview` file containing all the technical information about your project. This project file can be shared and opened via ASReview LAB. Via a [script](https://asreview.readthedocs.io/en/latest/example_api_asreview_file.html), you can access the content of the files.  The project API is for retrieving general project settings, the imported dataset, the feature matrix, etc. The state API retrieves data related directly to the reviewing process, such as the labels, the time of labeling, and the classifier used. In a [paper](https://psyarxiv.com/g93zf), we explain everything about what is stored from the process. 

The hundreds of thousands of lines of code behind ASReview LAB are available via [Github](https://github.com/asreview/asreview). Are you looking for something specific?
